example-events-client/utils.py

```diff
@@ -0,0 +1,171 @@
+import requests
+from typing import Optional
+
+
+def print_streaming_events(
+    events: list[dict],
+) -> None:
+    """Print all `CompletionChunkEvent` tokens.
+
+    Args:
+        events: The events to print.
+        printed_event_ids: The set of ids of already printed events. Used to prevent duplicate printing.
+    Returns:
+        The updated set of printed events.
+    """
+    completion_events = [
+        event for event in events if event["type"] == "CompletionChunkEvent"
+    ]
+
+    for completion_event in completion_events:
+        print(completion_event["payload"]["token"], flush=True, end="")
+
+
+def generate_headers(api_key: str) -> dict:
+    """Generates headers for Griptape Cloud API calls
+
+    Args:
+        api_key: a Griptape Cloud API Key. This is ignored when running the Skatepark emulator, but required for a Griptape Cloud hosted Structure.
+
+    Returns:
+        Dictionary for the header field in a requests call.
+    """
+    headers = {
+        "Authorization": f"Bearer {api_key}",
+        "Content-Type": "application/json",
+    }
+    return headers
+
+
+def create_structure_run(
+    host: str, api_key: str, structure_id: str, env: dict, args: list
+) -> dict:
+    """Create a Structure Run.
+
+    Args:
+        host: the host URL for the Structure.
+        api_key: a Griptape Cloud API Key. This is ignored when running the Skatepark emulator, but required for a Griptape Cloud hosted Structure.
+        structure_id: The Structure ID.
+        env: The environment variables to pass to the Structure program.
+        args: The arguments to pass to the Structure program.
+
+    Returns:
+        The created Structure Run.
+    """
+    response = requests.post(
+        f"{host}/api/structures/{structure_id}/runs",
+        json={"env": env, "args": args},
```

Comment:
the other client's utils.py is doing something different on this line. Maybe we have a single utils.py, and offer two clients in the same dir, instead of two separate ones?